test/e2e.js

```diff
@@ -8,19 +8,24 @@ var async = require('async');
 describe('meta endpoint', function() {
 
   var BASE_IFRAMELY_SERVER_URL = 'http://localhost:' + process.env.PORT;
-  var server = require('../server'); // start card meta
```

Comment:
These tests got a bit better. Since we control the app lifecycle we can start and shutdown the server for each test.